src/index.js

```diff
@@ -40,6 +40,57 @@ const _views = new WeakMap();
 
 let bypass = false;
 
+function nextOrPreviousProject (index) {
+    // Generate an array of projects by selecting children of 'project-viewer > ul[is="pv-list-tree"]'. For each 'li[is="pv-list-nested-item"]', expand it into its children.
+    const sidebar = document.getElementsByTagName('project-viewer')[0];
+    let list = sidebar.getElementsByTagName('ul')[0].children;
+
+    var output = [];
+
+    var expandProjects = function (element) {
+        if (element.getAttribute('is') === "pv-list-nested-item") {
+            let nestedList = element.getElementsByTagName('ul')[0].children;
```

Comment:
fixed and pushed.
@DamnedScholar this is only a working basis because i couldn't test due to the different DOM tree we have.

Do what you wan't with this information and let's kill this #22 !